console/src/components/Common/Layout/ExpandableEditor/Editor.js

```diff
@@ -29,7 +29,17 @@ class Editor extends React.Component {
   };
 
   toggleButton = () => {
-    if (this.props.isCollapsable === false && this.state.isEditing) {
+    const {
+      readOnlyMode = false,
+      isCollapsable,
+      service,
+      property,
+      collapseButtonText,
+      expandButtonText,
+    } = this.props;
+
+    const { isEditing } = this.state;
+    if (isCollapsable === false && isEditing) {
```

Comment:
```suggestion
    if (isCollapsable && isEditing) {
```

Reply:
this condition was already there in the previous code, do you want to toggle `isCollapsable` check and return null if it is true?

Reply:
or did you mean `if (!isCollapsable && isEditing) {`

Reply:
Yes, this is what I meant. Updated